src/media/css/base.styl

```diff
@@ -154,5 +154,42 @@ body {
 }
 
 #site-footer {
+    type--footer();
+    align-items: center;
     background: $color--greyscale--white;
+    display: flex;
+    justify-content: center;
+    padding: 20px;
+    
+    // Remove from the document flow and hide. Can't use display: none due to
+    // https://github.com/tj/nib/issues/291
+    position: absolute;
+    visibility: hidden;
+    +mq--show-footer() {
```

Comment:
Yeah, mq is short for media query, which was just too verbose. See:

https://github.com/mozilla/marketplace-style-guide/blob/master/src/media/css/constants/mq.styl
https://github.com/mozilla/marketplace-style-guide/blob/master/src/media/css/mq.styl